_test/export.response.test.php

```diff
@@ -12,31 +12,26 @@
 
 require_once(__DIR__ . '/../DokuwikiXMLExport.php');
 require_once(__DIR__ . '/../admin.php');
-require_once(__DIR__ . '/helper.php');
+require_once(__DIR__ . '/../_test/Helper.php');
 
 class export_response_test extends DokuWikiTest
 {
     public function setUp()
     {
-        $helper = new helper;
-        $helper->setUp();
+        Helper::setUp();
     }
 
     /**
      * @dataProvider parameterProviderForXMLResponse
      */
-    public function test_parameters_start_count_and_total_are_valid_for_($ids = array())
+    public function test_parameters_start_count_and_total_are_valid_for_($ids = array(), $expectedCount, $expectedTotal)
     {
-        $helper = new helper;
-        if ($ids) { // I also want to check for an empty DokuWiki
-            $helper->savePages(array($ids));
-            $expectedCount = count($ids);
-            $expectedTotal = count($ids);
-        } else {
-            $expectedCount = 0;
-            $expectedTotal = 0;
+        // If there are no pages, no pages can be saved.
+        if (!$expectedCount == 0 && !$expectedTotal == 0) {
+            Helper::savePages($ids);
```

Comment:
What happens if you pass an empty array to `savePages()`? Do you need the check before?